src/main/java/seedu/address/logic/commands/EditCommand.java

```diff
@@ -103,6 +103,9 @@ public CommandResult execute(Model model) throws CommandException {
         model.updateFilteredPersonList(PREDICATE_SHOW_ALL_PERSONS);
         model.getReminderList().setReminderListDirty();
 
+        // move focus to the edited person
+        model.getSelectedPerson().setValue(editedPerson);
```

Comment:
Nit: Perhaps it's better to move this whole `getSelectedPerson().setValule()` logic inside the model itself (e.g., `editSelectedPerson` or sth)?

Reply:
Yes, good idea!